chromium/rules.js

```diff
@@ -359,7 +359,7 @@ RuleSets.prototype = {
     var platform = ruletag.getAttribute("platform");
     if (platform) {
       default_state = false;
-      if (platform == "mixedcontent" && settings.enableMixedRulesets) {
+      if (platform === "mixedcontent" && settings.enableMixedRulesets) {
```

Comment:
I'm guessing that `==` and `===` have the same operator precedence?  Could you add parentheses here to make it more explicit anyway

Reply:
They do.